Rakefile

```diff
@@ -18,7 +18,7 @@ require_relative 'tasks/assets'
 require_relative 'tasks/profiles'
 require_relative 'tasks/geo'
 
-Mongoid.load!('./config/mongoid.yml')
+Mongoid.load!('./config/mongoid.yml', :test)
```

Comment:
This will always use the :test configuration for every rake task which is not what I want. I need to process `rake db:seed` in production, too.